linux_os/guide/system/permissions/partitions/mount_option_nodev_nonroot_local_partitions/tests/correct.pass.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+. $SHARED/partition.sh
+
+umount --all
```

Comment:
What is the purpose of this line? Currently, it renders system kinda unusable and tests can't continue.

Reply:
To ensure that everything is unmounted except partitions that are specified below. Idea is to have "clean" environment for testing and not to specify them individually.

Reply:
When I remove `umount --all` the tests are executed, although they fail.